### ISSUE TITLE
[BUG] Cannot generato self-signed certificate

### ISSUE THREAD

**Issue by Sumi-Sumi** [bunkerity/bunkerized-nginx#142]
**Description**
When I run using self.signed certificate, a container does not generate it.
It seems that the variables are not being passed to open-ssl properly and the self-certificate is not being generated.

**How to reproduce**

```
podman run
 -it
 --rm
 -p 8000:8080
 -p 8443:8443
 -e GENERATE_SELF_SIGNED_SSL=yes
 -e SELF_SIGNED_SSL_EXPIRY=100
 -e REDIRECT_HTTP_TO_HTTPS=yes
 -e REVERSE_PROXY_URL=/
 -e REVERSE_PROXY_HOST=*******
 -e USE_REVERSE_PROXY=yes
 docker.io/bunkerity/bunkerized-nginx:latest 
```

**Logs**
```
[*] Starting bunkerized-nginx ...
[*] Configuring bunkerized-nginx ...
Ignoring container=podman
Ignoring TERM=xterm
Ignoring SELF_SIGNED_SSL_EXPIRY=100
[*] Generation done !
[!] Error while generating self-signed certificate : req: Can't parse "-subj" as a number
req: Non-positive number "-subj" for -days
req: Use -help for summary.
[*] Generated self-signed certificate for default server
[*] Downloading bad user-agent list (in background) ...
[*] Downloading bad referrer list (in background) ...
[*] Downloading tor exit nodes list (in background) ...
[*] Downloading proxies list (in background) ...
[*] Downloading abusers list (in background) ...
[*] Running nginx ...
crond[118]: crond (busybox 1.32.1) started, log level 8
crond[118]: root: Permission denied
2021/06/10 12:45:15 [emerg] 119#119: invalid number of arguments in "ssl_certificate" directive in /etc/nginx/https.conf:4
nginx: [emerg] invalid number of arguments in "ssl_certificate" directive in /etc/nginx/https.conf:4
[*] bunkerized-nginx stopped
```

**Comment by bunkerity** [comment]
Hello @Sumi-Sumi,

Thanks for reporting it. I've added a fix into the dev branch, will keep you updated when it's integrated into the next version.

**Comment by bunkerity** [comment]
Hey @Sumi-Sumi,

I've added the fix to the v1.2.7. Feel free to open a new issue if needed. Thanks again.
